### DISCUSSION TITLE
Can't download songs or log in.

### DISCUSSION THREAD

**Discussion by Schoology-Assignment** [ppy/osu#23322] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

Whenever I go and try to download the "first-run setup" songs it says "Beat Map download failed" and so I decided just to download songs on the browse page but it says I need to log in, but when I try to log in it says "The SSL connection could not be established see inner exception", so right now I can't play any songs on osu. When I try to make a new account it says "Something Happened... but we're not sure what"

### Screenshots or videos

![image](https://user-images.githubusercontent.com/83793295/234715587-bb0e9940-eb1f-4b31-904d-2429fd6576a2.png)
![image](https://user-images.githubusercontent.com/83793295/234716258-e174121e-d6c1-47ba-a95a-976615816eab.png)

### Version

2023.419.0

### Logs

[input.log](https://github.com/ppy/osu/files/11337483/input.log)
[network.log](https://github.com/ppy/osu/files/11337484/network.log)
[performance.log](https://github.com/ppy/osu/files/11337485/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11337486/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11337487/updater.log)
[database.log](https://github.com/ppy/osu/files/11337488/database.log)

**Comment by megadrive7** [comment]
Well first off DO NOT MAKE ANOTHER ACCOUNT. It's a bannable offense. Hope you figure it out though

**Reply by Schoology-Assignment** [reply]
I know I just did not what else to do.

**Comment by peppy** [comment, marked as answer]
This usually means your whole system's SSL chain is broken in some way.

Try searching google for more information.

**Reply by pwtoothless** [reply]
I'm having the same issue but i cant seem to fix it.
